compiler/damlc/BUILD.bazel

```diff
@@ -273,7 +273,7 @@ genrule(
             --template=$(location :daml-base-rst-template) \\
             --index-template=$(location :daml-base-rst-index-template) \\
             --hoogle-template=$(location :daml-base-hoogle-template) \\
-            --base-url=https://docs.daml.com/daml/stdlib \\
+            --base-url=https://docs.daml.com/daml-script/api \\
```

Comment:
This seems wrong to me. Is this the intended result for functions in the stdlib?
```
@url https://docs.daml.com/daml-script/api/GHC-Stack-Types.html#function-ghc-stack-types-emptycallstack-65809
emptyCallStack :: CallStack
```

Reply:
Good catch, fixed! Not quite sure how that happened.

Reply:
Oh, that sounds bad. The `--base-url` should be the url for the generated documentation (i.e. this option should be correct). To get correct URLs for stdlib/prim you should feed the anchor table from the stdlib documentation (via `--input-anchor`)

Reply:
We should absolutely use anchors but I noticed that I cannot actually have multiple anchors which I need for at least triggers so I’ll leave the whole anchor business to a separate PR.